docs/docs/reach-router-and-gatsby.md

```diff
@@ -11,6 +11,7 @@ The main reasons Gatsby uses `@reach/router` are:
 1. Preloading. You can read more about preloading in the docs for the [Gatsby Link API](https://www.gatsbyjs.org/docs/gatsby-link/).
 2. The [routing accessibility](https://reach.tech/router/accessibility) it provides.
 3. It supports [server rendering](https://reach.tech/router/server-rendering) which helps Gatsby build routed files at build time.
+4. Scroll restoration.
```

Comment:
This should probably be a full sentence with a link to learn more, like the other list items.

Reply:
I like this change, but I don't really know what to link to. Maybe this? https://reacttraining.com/react-router/web/guides/scroll-restoration

Reply:
Sorry! Yes, that's what I meant by "Reach docs". That was, in no way, clear. 😳

Reply:
fwiw that's react router docs. Reach doesn't have a section on scroll restoration

Reply:
That... is true. It _is_ the page I was thinking of, regardless. 😅